ydb/core/protos/config.proto

```diff
@@ -1511,6 +1511,25 @@ message TCompactionConfig {
     optional TBorrowedCompactionConfig BorrowedCompactionConfig = 2;
 }
 
+message TBackgroundCleaningConfig {
+    optional double MaxRate = 1 [default = 0]; // unlimitted
+    optional uint64 InflightLimit = 2 [default = 10]; // TODO: consider more?
+
+    // After this interval we will try to restart
+    optional uint64 TimeoutSeconds = 3 [default = 15];
```

Comment:
We should reduce the complexity of the configuration a bit. It would be more accurate to say that the temporary table is deleted within 300 seconds of the node's disconnect. It's still acceptable if it is deleted after 350 seconds but it would be preferable to have a lower bound specified.

So, let's introduce TemporaryTableRemovalTimeout which by default will be 300 seconds.

Reply:
https://github.com/ydb-platform/ydb/pull/513/files#diff-2e781bd76ab9922cc0012cb685cbf5adcc215185dd89fb3b9f2566f407c62f3bR1530

Reply:
We will configure this parameter through the TRetrySettings.